src/scripts/main.js

```diff
@@ -2,7 +2,22 @@
 
 const people = require('./lib/people');
 
-// eslint-disable-next-line no-console
-console.log(people); // you can remove it
+people.map(person => {
+  person.age = person.died - person.born;
+  person.century = Math.ceil(person.died / 100);
+});
```

Comment:
better to do it right in next `for`.

it's not a good idea to change input data, also you should set the results of `.map` to a variable if you want to use it